### DISCUSSION TITLE
Assign issue to multiple assignees

### DISCUSSION THREAD

**Discussion by lauggh** [community/community#7150] [Projects and Issues] [answered]
Was trying to assign an issue to multiple assignees. It would flash to show user1 and user2 but then it would settle on just showing the last assigned user in the list. 

Then went to the issue detail screen to try to assign multiple users and it also assigned and reassigned users... Unable to assign multiple users to an issue. This is confusing since I was able to do this on Issues before.

**Comment by FranzSw** [comment, marked as answer]
This depends on the pricing level that you are on. However I agree that the communication could be improved on the project / board screen.

<img width="1303" alt="image" src="https://user-images.githubusercontent.com/15076254/139955205-120a20af-78ed-49a8-9247-7e1ddf4a25ca.png">

**Reply by lauggh** [reply]
ah, thank you for this. I did not realize that it was a paid feature. I have both public and private repos so it was confusing to me to see this available on some but not on others.
